docs/README.md

```diff
@@ -1,4 +1,4 @@
 README
 ======
 
-This file is a placeholder. This directory will contain public documentation.
+This file is a placeholder.  This directory will contain public documentation.
```

Comment:
:ok_hand: 

Two spaces after a period is fine in markdown. It improves readability in the raw markdown but will *still be rendered as a single space by browsers after being converted to HTML*.

Miguel's [Markdown Style Guide](http://tirania.org/blog/archive/2014/Sep-30.html).